### ISSUE TITLE
Version without Code Contracts

### ISSUE THREAD

**Issue by NickStrupat** [xperiandri/StringBuilderExtensions#1]
Hi!

Myself and many others have trouble installing Code Contracts, especially with VS2015 and likely the upcoming VS2017.

Would you be interested in releasing another version of your extensions without the dependency on Code Contracts?

**Comment by xperiandri** [comment]
I can make a release NuGet for you.
I already have a pre-release package on NuGet. Have you found it?

**Comment by NickStrupat** [comment]
Yes, I've tried the NuGet package. Thanks, it is fine as far as I can tell. It's just the Code Contracts tools that I can't get to work on my machine.

I'll see if I can make the changes right now over my lunch break. I'll fork and do a pull-request :)

**Comment by NickStrupat** [comment]
I was able to refactor out the `Contract.Requires` called into checks, and some of the `Contract.Ensures` (most of them remain in my current fork). I put up a package called `NickStrupat.StringBuilderExtensions.NoCodeContracts`. I'll use that for now since I'm in a bit of a rush, but I'd be happy to figure out what you'd like to do and defer to that since you started the project :1st_place_medal:

**Comment by xperiandri** [comment]
@NickStrupat, I've found a package `CodeContracts.MSBuild` and now it builds well
